s3prl/downstream/asr/hf_hub_templates/README.md

```diff
@@ -0,0 +1,15 @@
+---
+library_name: superb
+benchmark: superb
+task: asr
+datasets:
+- superb
```

Comment:
i also added the name of the dataset so we can link back all the fine-tuned models